openjdk.test.load/config/inventories/mauve/mauve_all_exclude.xml

```diff
@@ -1,6 +1,8 @@
 <inventory>
+	<!-- Excluded as invalid test. Ref: https://github.com/eclipse/openj9/issues/7020 -->
+	<mauve class="gnu.testlet.java.io.InputStreamReader.utf8"/>
```

Comment:
Can the test in question within `mauve_multiThread_exclude.xml` be removed assuming this covers both single/multi thread mode?

Reply:
The "all" part may be a bit confusing :) , but the [way](mauve_multiThread_exclude.xml) STF framework is designed, excludes in `mauve_multiThread_exclude.xml` would only apply to load tests that use mauve_multiThread.xml configuration (e.g. [MauveMultiThreaded](https://github.com/adoptium/aqa-systemtest/blob/master/openjdk.test.load/src/test.load/net/adoptopenjdk/stf/MauveMultiThrdLoad.java#L45) test). 

The test in question uses [mauve_all.xml](https://github.com/adoptium/aqa-systemtest/blob/master/openjdk.test.load/src/test.load/net/adoptopenjdk/stf/MauveSingleInvocLoad.java). And to exclude a sub-tests for this we'd have to do what was done in this PR.